notebooks/Pyterrier.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
You could have added some Comments / Markdown cells to explain what you are doing. This way it is a little bit hard to understand at first glance.